core/common/src/main/java/alluxio/security/authentication/DefaultAuthenticationServer.java

```diff
@@ -125,6 +125,13 @@ public void unregisterChannel(UUID channelId) {
         LOG.warn("Failed to dispose sasl client for channel-Id: {}. Error: {}", channelId,
             e.getMessage());
       }
+
+      try {
+        channelInfo.getSaslServerDriver().onCompleted();
+      } catch (Exception e) {
+        LOG.warn("Failed to authentication session for channel-Id: {}. Error: {}", channelId,
```

Comment:
NIT: what does this message mean?

Reply:
Oh. It's gotta be "Failed to **complete** the authentication ......"